pkg/executor/set_test.go

```diff
@@ -1351,14 +1351,6 @@ func TestValidateSetVar(t *testing.T) {
 	err = tk.ExecToErr("set @@global.innodb_ft_enable_stopword=2")
 	require.True(t, terror.ErrorEqual(err, variable.ErrWrongValueForVar), fmt.Sprintf("err %v", err))
 
-	tk.MustExec("set @@query_cache_type=0")
-	result = tk.MustQuery("select @@query_cache_type;")
-	result.Check(testkit.Rows("OFF"))
-
-	tk.MustExec("set @@query_cache_type=2")
-	result = tk.MustQuery("select @@query_cache_type;")
-	result.Check(testkit.Rows("DEMAND"))
-
```

Comment:
Maybe update this to `tk.MustContainErrMsg()` or so, instead of removing it?

Reply:
done in 4274325b70f9d31f37ff3192a8171ac99b409882